dashboard/app/controllers/api/v1/teacher_feedbacks_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Api::V1::TeacherFeedbacksController < Api::V1::JsonApiController
+  load_and_authorize_resource
```

Comment:
It appears the only action that's using these pre-loaded resources is `create`, so there's no need to load for all actions. Remove the `load_and` part and just call `authorize_resource` here, then add the following line to load only for that one action:
```ruby
load_resource, only: :create
```